packages/react-components/react-alert/src/components/Alert/useAlertStyles.ts

```diff
@@ -1,34 +1,78 @@
-import { makeStyles, mergeClasses } from '@griffel/react';
+import { tokens } from '@fluentui/react-theme';
+import { makeStyles, mergeClasses, shorthands } from '@griffel/react';
+
 import type { AlertSlots, AlertState } from './Alert.types';
 import type { SlotClassNames } from '@fluentui/react-utilities';
 
 export const alertClassName = 'fui-Alert';
 export const alertClassNames: SlotClassNames<AlertSlots> = {
   root: 'fui-Alert',
-  // TODO: add class names for all slots on AlertSlots.
-  // Should be of the form `<slotName>: 'fui-Alert__<slotName>`
+  icon: 'fui-Alert__icon',
+  action: 'fui-Alert__action',
 };
 
-/**
- * Styles for the root slot
- */
 const useStyles = makeStyles({
   root: {
-    // TODO Add default styles for the root element
+    display: 'flex',
+    alignItems: 'center',
+    minHeight: '44px',
+    ...shorthands.padding('0', '12px'),
+    backgroundColor: tokens.colorNeutralBackground1, // todo - there is no bg10, used bg1
+    ...shorthands.borderColor('transparent'),
+    ...shorthands.borderRadius('4px'),
+    boxShadow: tokens.shadow8,
+    fontSize: tokens.fontSizeBase300, // todo - lineheight in tokens
+    fontWeight: tokens.fontWeightSemibold,
   },
+  icon: {
+    height: '16px',
```

Comment:
```suggestion
    height: '16px',
    fontSize: '16px',
```

`fontSize` might be required for scalable icons, but please check first.